config/config.go

```diff
@@ -218,11 +219,17 @@ type HDFSArtifactRepository struct {
 	Force bool `json:"force,omitempty"`
 }
 
-// PrometheusConfig defines a config for a metrics server
-type PrometheusConfig struct {
-	Enabled       bool   `json:"enabled,omitempty"`
-	DisableLegacy bool   `json:"disableLegacy"`
-	MetricsTTL    TTL    `json:"metricsTTL"`
-	Path          string `json:"path,omitempty"`
-	Port          string `json:"port,omitempty"`
+// MetricsConfig defines a config for a metrics server
+type MetricsConfig struct {
+	// Enabled controls metric emission. Default is true, set "enabled: false" to turn off
+	Enabled *bool `json:"enabled,omitempty"`
+	// DisableLegacy turns off legacy metrics
+	// DEPRECATED: Legacy metrics are now removed
+	DisableLegacy bool `json:"disableLegacy"`
```

Comment:
Removed legacy metrics

Reply:
I think you can actually delete this field.

Reply:
I chose to leave it because it documents a change. Users who are used to the field might wonder why it disappeared. What do you think?

Reply:
OK. Can you state the field is ignored?

Reply:
Done